src/components/fillProfileForm/cardInfoCard/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useTranslation } from 'react-i18next';
+import {
+  StyledContainer,
+  StyledRow,
+  StyledSmallInput,
+} from 'components/fillProfileForm/cardInfoCard/styles';
+import {
+  StyledCard,
+  StyledFormContainer,
+  StyledInput,
+  StyledSubtitle,
+  StyledTitle,
+} from 'components/fillProfileForm/sharedStyles';
+
+const expDatePH = 'MM/YYYY';
+const cvvPH = 'CVV';
+
+const CreditCardForm = () => {
```

Comment:
we cant create own form its not secure. we need to use stripe form, or other payment provider form

Reply:
please give some answer, not clear for me what you decide to do with it

Reply:
I decided to use stripe form for it since we will use it for payments later